.github/workflows/master.yml

```diff
@@ -14,6 +14,13 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.6.x'
+
+      - name: Build requirements
+        run: pip3 install "setuptools>=45" wheel
```

Comment:
This should be added to test requirements instead.

Reply:
Smart! 👍